docs/real-world-examples/pneumonia.md

```diff
@@ -0,0 +1,54 @@
+
+> For the sake of simplicity, we will only provision an _open_ setup. Do not upload sensitive data to it! 
+
+## Prerequisites
+To enjoy this tutorial, you will need to:
+- have an active [Azure subscription](https://azure.microsoft.com) that you can use for development purposes,
+- have permissions to create resources, set permissions, and create identities in this subscription (or at least in one resource group),
+  - Note that to set permissions, you typically need _Owner_ role in the subscription or resource group - _Contributor_ role is not enough. This is key for being able to _secure_ the setup.
+- [install the Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli),
+- have a way to run bash scripts;
+- **fork** this repository (_fork_ as opposed to _clone_ because you will need to create GitHub secrets and run GitHub actions to prepare the data).
+
+## Procedure
+The procedure to run this example  has three phases:
+- provision the Azure resources;
+- prepare the data;
+- run the job.
+
+Each phase is described in the subsections below.
+
+### Provisioning
+This phase is easy. Just follow the instructions in the [quickstart](../quickstart.md) to provision an open sandbox. Make note of the name of the resource group you provisioned, as well as the name of the workspace.
+
+### Data preparation
+The goal of this phase is to upload to Azure ML 3 distinct datasets (one per silo), each subdivided into train, test, and validation. This is achieved by running a GitHub action that will download the dataset from [Kaggle](https://www.kaggle.com/datasets/paultimothymooney/chest-xray-pneumonia), split it evenly between the silos, and upload the 3 data assets. We will need to first create an Azure Service Principal with enough permissions to create the data assets in Azure ML. The whole procedure is explained below, and closely follows [the one](https://github.com/Azure/medical-imaging/blob/main/federated-learning/README.md#1-prepare-your-experiment) developed by Harmke Alkemade _et al_ .
+
+1. Create a service principal with contributor access to your resource group by running the following Azure CLI command.
+```bash
+  az ad sp create-for-rbac --name "<service-principal-name>" --role contributor --scopes /subscriptions/<subscription-id>/resourceGroups/<your-resource-group-name> --sdk-auth
+```
+2. Create a GitHub secret in your _forked_ repository. Name it AZURE_CREDENTIALS_DATAPREP and copy and paste the output of the above command to the Value field of the secret.
```

Comment:
This whole process applies to Harmke's repository but I don't think this process should apply in our case. I would rather give our users some job to run than have to mess up with github secrets etc.

Reply:
Feel free to push only the pipeline part of your job, that assumes the data is in a given location in each silo, then you can push the job to upload the data in a distinct PR

Reply:
People will have to deal with secrets no matter what, as long as we're using this kaggle dataset. 

I'd rather put everything in a single PR

Reply:
Hmmm, actually with the uncertainty about how to do the data provisioning, I'd rather get this in soon, then do a follow-up the data provisioning